frontend/views/containers/settings/UserProfile.vue

```diff
@@ -143,22 +144,23 @@ export default {
       },
       ephemeral: {
         errorMsg: null,
+        newPicture: false,
         profileSaved: false
       }
     }
   },
   validations: {
     form: {
-      picture: { url },
+      picture: {},
       email: { email }
     }
   },
   computed: {
     attributes () {
       return this.$store.getters.currentUserIdentityContract.attributes || {}
     },
-    userPicture () {
-      return this.form.picture || this.attributes.picture
+    userPictureInitial () {
+      return this.attributes.picture
```

Comment:
Is it really necessary for this computed property to exist, or can you simply bind Avatar's `:src` to `attributes.picture`?

Reply:
I believe so. We can't bind `:src` to `attributes.picture`, otherwise it would read the value directly from this `this.form.picture = fileList[0]` and that's a File, not a String as Avatar needs.

Reply:
What? Why?